modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -192,7 +198,106 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_worker-machi
 ----
 +
 Because you create and manage the worker machines yourself, you do not need to initialize these machines.
+//The current AWS UPI method removes all of the `99_openshift-cluster-api_worker-machineset-* files. What is different about the Local Zones that means that we should retain them? Should we remove whatever the installation program creates and then specifically create this new file?
 endif::aws,azure,ash,gcp[]
+ifdef::localzone[]
+. Create the machine set manifests for the worker nodes in your Local Zone.
+.. Export a variable to define the instance type for the worker machine to deploy on the Local Zone subnet by running the following command:
++
+[source,terminal]
+----
+$ export INSTANCE_TYPE="<instance_type>" <1>
+----
+<1> Set `<instance_type>` to a tested instance type, such as `c5d.2xlarge`.
```

Comment:
If it is possible, we could add a helper command to help the user to check which instances are available on the Local Zone he is deploying (it can change by Local Zone location). For example:

---

To check the available instance types in the Location you are deploying, run the following command - using the location `us-east-1-nyc-1a`:
```
aws ec2 describe-instance-type-offerings \
    --location-type availability-zone \
    --filters Name=location,Values=us-east-1-nyc-1a \
    --region us-east-1
```